recipes/DAVE/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "useDAVE" %}
+{% set version = "0.1.0" %}                         
+package:
+  name: dave
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e3ca64989027fa7ebcb7a346864f2e84a5fcf90106aec897b774be1111d6b32a
+ 
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
The whole build section can be omitted, this is only needed when you need compilers.